src/views/navbar/style.js

```diff
@@ -197,6 +199,16 @@ export const Logo = styled(Tab)`
   @media (max-width: 768px) {
     display: none;
   }
+
+  ${props =>
+    props.isHidden &&
+    css`
+      position: absolute;
+      width: 1px;
+      height: 1px;
+      overflow: hidden;
+      clip: rect(1px, 1px, 1px, 1px);
+    `};
```

Comment:
I guess we could re-use the `VisuallyHidden` component styles instead?

Reply:
@Tom-Bonnike But this is the logo, right? So it needs to be shown visually, hence we cannot use `display: none`, we just need to prevent the focus (as you pointed), so using `aria-hidden` is fine.

Reply:
Yah, but these styles are here to hide it (from all users). So `display: none` is fine. :)

Reply:
Ah, I haven't notice the `props.isHidden` prop :)